tests/app/questionnaire/routing/test_operators.py

```diff
@@ -0,0 +1,283 @@
+from datetime import datetime, timedelta
+
+import pytest
+from dateutil.relativedelta import relativedelta
+
+from app.questionnaire.routing.helpers import datetime_as_midnight
+from app.questionnaire.routing.operator import Operator
+from app.questionnaire.rules import convert_to_datetime
+
+now = datetime.utcnow()
+now_as_yyyy_mm_dd = now.strftime("%Y-%m-%d")
+now_as_yyyy_mm = now.strftime("%Y-%m")
+now_as_yyyy = now.strftime("%Y")
+
+test_data_equals_operation_numeric_and_date_matching_values = [
+    [(0.5, 0.5), True],
+    [(1.0, 1), True],
+    [(3, 3), True],
+    [(now, now), True],
+]
```

Comment:
I think the same comment made on the value sources pr applies here too. The test data is too far removed from the tests and it makes them hard to read. We should consider some duplication to make the tests more readable.

Reply:
Duplicated test date to improve readability in both this file and the rules file also.